ts/live-compositor/src/components/InputStream.ts

```diff
@@ -12,9 +14,28 @@ export type InputStreamProps = {
    * Id of an input. It identifies a stream registered using a `LiveCompositor.registerInput`.
    */
   inputId: Api.InputId;
+  /**
+   * Audio volume represented by a number between 0 and 1.
+   */
+  volume?: number;
+  /**
+   * Mute audio.
+   */
+  mute?: boolean;
```

Comment:
Why do you add `mute`? Can't they just set `volume=0` if somebody wants to mute audio? Having 2 separate fields for that doesn't make sense IMO. It also introduces ambiguity when both values are specified.

Reply:
> Why do you add mute? 

I think it's a useful API, especially in React you can do `<InputStream mute/>`. I also considered using `muted` instead (I think mute is better, but muted is the same as in `<video/>` HTML tag), but I don't have strong opinions on that.

> Can't they just set volume=0

Yes, as you can see below it's literally what it does internally

> Having 2 separate fields for that doesn't make sense IMO.

It's often true, but I disagree with it as a general statement. This is a tradeoff, we are getting simpler API to do sth specific, but increase the complexity because user needs to know how those 2 ways interact. You can argue if you want that tradeoff is not worth it here, but not in general.

> It also introduces ambiguity when both values are specified.

Does it? Can you imagine someone seeing those 2 options and assuming that volume has higher priority than mute?